### ISSUE TITLE
Idris doesn't compile with transformers-0.4.2.0

### ISSUE THREAD

**Issue by suhr** [idris-lang/Idris-dev#2338]
Output:
```
[58 of 98] Compiling Idris.Output     ( src/Idris/Output.hs, dist/build/Idris/Output.o )

src/Idris/Output.hs:28:10:
    Could not deduce (transformers-0.4.2.0:Control.Monad.IO.Class.MonadIO
                        (ExceptT Err m))
      arising from the superclasses of an instance declaration
    from the context (MonadException m)
      bound by the instance declaration at src/Idris/Output.hs:28:10-59
    In the instance declaration for ‘MonadException (ExceptT Err m)’
```

GHC version: 7.10.1

**Comment by david-christiansen** [comment]
Sorry to be slow on responding to this! I've been a bit busy. I'll take a look and see if we need to tweak code or bounds.

**Comment by suhr** [comment]
Fixed with update to 7.10.2.
